avalon/houdini/pipeline.py

```diff
@@ -175,16 +172,18 @@ def containerise(name,
 
     """
 
-    # Get main object network node
-    obj_network = hou.node("/obj")
+    # Ensure AVALON_CONTAINERS subnet exists
```

Comment:
Refactored code to take in account the new non-relative path of `AVALON_CONTAINERS` - also matched naming of containers somewhat more with how Maya was doing things. _Basically solely cosmetic changes_